### ISSUE TITLE
Ruby Dependency Issue

### ISSUE THREAD

**Issue by grycyk** [fastlane/fastlane-plugin-firebase_test_lab#58]
Hey,

I am moving our testing over to Firebase and set up our Fastlane with with plugin the backend of last week, but looks like there is is dependency issues.

We are running `fastlane_version "2.113.0"`

The error when running the lane for Firebase is below:

```Error loading plugin 'fastlane-plugin-firebase_test_lab': Unable to activate fastlane-plugin-firebase_test_lab-1.0.2, because google-cloud-storage-1.15.0 conflicts with google-cloud-storage (~> 1.13.0)```

Myself and one of our devs have looked at many different ways to solve this, but to no avail.

Any chance this can be looked into?

Cheers,

Grae

**Comment by MartyCatawiki** [comment]
@grycyk Dependency `google-cloud-storage` is now updated in latest release 1.0.3, see #59

**Comment by grycyk** [comment]
@MartyCatawiki just a heads up seeing a similar issue again this time with ```Fastlane 2.116.0``` 

but now with, slightly different version issue, see below

```Error loading plugin 'fastlane-plugin-firebase_test_lab': Unable to activate fastlane-plugin-firebase_test_lab-1.0.4, because google-cloud-storage-1.16.0 conflicts with google-cloud-storage (~> 1.15.0)```

**Comment by grycyk** [comment]
We are looking into this right now (myself and another one of our devs), to find a fix for us right now, but not sure what happening yet. If we find out something i'll post our findings.

**Comment by dungntm-teko** [comment]
I got it too, did you find anything to solve? @grycyk

**Comment by mschinis** [comment]
any updates on this?

**Comment by ravimaurya-nickelfox** [comment]
Is there any update??

**Comment by khomchyk** [comment]
Any updates?

**Comment by Ravi448** [comment]
> We are looking into this right now (myself and another one of our devs), to find a fix for us right now, but not sure what happening yet. If we find out something i'll post our findings.

Is there any update?

**Comment by MartyCatawiki** [comment]
@Ravi448 I think this question should be asked to @grycyk . And @powerivq is the maintainer of this project, i hope he can help or make ownership to the others

**Comment by grycyk** [comment]
Just ran into this issue again: 

Error loading plugin 'fastlane-plugin-firebase_test_lab': Unable to activate fastlane-plugin-firebase_test_lab-1.0.5, because google-cloud-storage-1.25.1 conflicts with google-cloud-storage (~> 1.15.0)

AFAIK we never found a solution to this, but I to clarify my comment @MartyCatawiki & @Ravi448 (quote below) I meant to say was we were looking for a work around to this issue, but we never did and I have ran into this again today.

> We are looking into this right now (myself and another one of our devs), to find a fix for us right now, but not sure what happening yet. If we find out something i'll post our findings.

**Comment by Ravi448** [comment]
@grycyk 
Follow below steps to resolve the issue

1. Install plugin using my forked repo : [https://github.com/Ravi448/fastlane-plugin-firebase_test_lab](url)
2. run `fastlane install_plugins` to make sure everything is installed correctly, If this command fails then run `bundle exec fastlane install_plugins`
3. Run `bundle exec fastlane firebase_test`

If you are using CI/CD to build, please make sure to commit Gemfile and Gemfile.lock

It'll work.

**Comment by grycyk** [comment]
Not sure if it will fix it but decided to make a PR to solve this: #70 if this works it should close #69 also.

**Comment by Ravi448** [comment]
@grycyk It'll work for you I believe. It worked for me!

**Comment by grycyk** [comment]
I see you just removed the dependency instead of a version bump, that what I have done in that PR.

**Comment by farhanf** [comment]
Bump. Anyone still getting this issue? It's conflicting with a newer version of GCS. 

`Error loading plugin 'fastlane-plugin-firebase_test_lab': Unable to activate fastlane-plugin-firebase_test_lab-1.0.5, because google-cloud-storage-1.25.1 conflicts with google-cloud-storage (~> 1.15.0)`

**Comment by grycyk** [comment]
afaik my PR has fixed this, we just need a version bump of the plugin but perhaps you can install this from my branch??

**Comment by farhanf** [comment]
You're right that did fix it. Ended up cloning and applying the same fix you did. Using fastlane from inside Gitlab CI runner so I have to provide gems with local path anyway.

Now my issue is that the build products (zip file) from `fastlane scan` with the `build_for_testing` flag is not being accepted by Test Lab. But if I upload the same zip using the Firebase console GUI it works and performs the test? Not sure what the issue is here, any guidance appreciated.

**Comment by billibala** [comment]
Current version of Fastlane (2.185.1) requires google-cloud-storage (~> 1.31)

```
Bundler could not find compatible versions for gem "google-cloud-storage":
  In Gemfile:
    fastlane was resolved to 2.185.1, which depends on
      google-cloud-storage (~> 1.31)

    fastlane-plugin-firebase_test_lab was resolved to 1.0.2, which depends on
      google-cloud-storage (~> 1.13.0)
```

**Comment by MartyCatawiki** [comment]
> Current version of Fastlane (2.185.1) requires google-cloud-storage (~> 1.31)
> 
> ```
> Bundler could not find compatible versions for gem "google-cloud-storage":
>   In Gemfile:
>     fastlane was resolved to 2.185.1, which depends on
>       google-cloud-storage (~> 1.31)
> 
>     fastlane-plugin-firebase_test_lab was resolved to 1.0.2, which depends on
>       google-cloud-storage (~> 1.13.0)
> ```

@billibala The guys already changed it on master, but still not published a new version yet. See [this ](https://github.com/fastlane/fastlane-plugin-firebase_test_lab/issues/74)issue.

**Comment by nhhai** [comment]
Is this released? I'm having the same issue now, it seems the file on master was updated but when I use this command `fastlane add_plugin firebase_test_lab`, the version it installed still has this problem.
